vnext/Shared/Modules/UIManagerModule.cpp

```diff
@@ -248,20 +249,24 @@ void UIManager::createView(
     std::string &&className,
     int64_t /*rootViewTag*/,
     folly::dynamic && /*ReadableMap*/ props) {
-  m_nativeUIManager->ensureInBatch();
-  auto viewManager = GetViewManager(className);
-  auto node = viewManager->createShadow();
-  node->m_className = std::move(className);
-  node->m_tag = tag;
-  node->m_viewManager = viewManager;
-
-  node->createView();
-  m_nativeUIManager->CreateView(*node, props);
-
-  m_nodeRegistry.addNode(shadow_ptr(node), tag);
-
-  if (!props.isNull())
-    node->updateProperties(std::move(props));
+  try {
+    m_nativeUIManager->ensureInBatch();
+    auto viewManager = GetViewManager(className);
+    auto node = viewManager->createShadow();
+    node->m_className = std::move(className);
+    node->m_tag = tag;
+    node->m_viewManager = viewManager;
+
+    node->createView();
+    m_nativeUIManager->CreateView(*node, props);
+
+    m_nodeRegistry.addNode(shadow_ptr(node), tag);
+
+    if (!props.isNull())
+      node->updateProperties(std::move(props));
+  } catch (winrt::hresult_error &hr) {
+    throw Microsoft::Common::Unicode::Utf16ToUtf8(hr.message());
```

Comment:
>throw [](start = 4, length = 5)

It is considered as an anti-pattern in the modern C++ to throw simple types. The recommended approach is to always throw exceptions inherited from the std::exception.
Why is it needed here?